src/sentry/static/sentry/app/components/assistant/guideAnchor.jsx

```diff
@@ -1,10 +1,11 @@
 import {ClassNames} from '@emotion/core';
 import PropTypes from 'prop-types';
 import React from 'react';
-import styled from '@emotion/styled';
 import $ from 'jquery';
 import createReactClass from 'create-react-class';
 import Reflux from 'reflux';
+
+import styled from '@emotion/styled';
```

Comment:
This is wrong :( though technically somewhat true due to the webpack config. Maybe that's how this works?

Reply:
we have a alias for @emotion/styled .. unfortunately I don't know the reason behind it 
![image](https://user-images.githubusercontent.com/29228205/73792778-4ae63c00-47a5-11ea-9f6c-6567c76e1be7.png)

Reply:
It's because we have to re-export it with our own type, that is why.

Btw you can link directly to code in github like this:

https://github.com/getsentry/sentry/blob/d6925547324b4dbaf51b4eb97aed836c65b5b6e0/webpack.config.js#L327-L334

Reply:
Oh nice, I'll do next time 😉 Thanks for the tip

Reply:
@EvanPurkhiser about what you said:

> It's because we have to re-export it with our own type

So the name must be somehow related to Sentry, otherwise it is very confusing ..

Reply:
@billyvg Any thoughts on giving the emotion export a name like `app/emotion-styled`. I'm actually not positive we're able to do that?

Reply:
I think we can do that

Reply:
done.